Basic-Car-Maintenance/Shared/Localizable.xcstrings

```diff
@@ -1471,7 +1503,7 @@
       }
     },
     "Select a vehicle" : {
-      "comment" : "Maintenance event vehicle picker label"
+      "comment" : "Maintenance event vehicle picker label\nOdometer reading vehicle picker label\nOdometer Reading vehicle picker label"
```

Comment:
Something I recently discovered, is that because the comments are auto-generated from the string key, that's used, all of the comment should be the same for the same string key.

See this diff, and you can find the code where this is used. Redo the comments so that it's more general, but still descriptive, if possible.